.bbtravis.yml

```diff
@@ -49,12 +49,9 @@ matrix:
     - env: PYTHON=3.9 TWISTED=latest SQLALCHEMY=latest TESTS=interop WORKER_PYTHON=3.7
     - env: PYTHON=3.9 TWISTED=latest SQLALCHEMY=latest TESTS=interop WORKER_PYTHON=3.6
     - env: PYTHON=3.9 TWISTED=latest SQLALCHEMY=latest TESTS=interop WORKER_PYTHON=3.5
-    - env: PYTHON=3.9 TWISTED=latest SQLALCHEMY=latest TESTS=interop WORKER_PYTHON=2.7
```

Comment:
For interoperability tests we should still test that old Buildbot worker can connect to newer master. We can simply install buildbot worker from pip. It would make sense to add PIP_WORKER_VERSION and install that package instead of installing from the git checkout.

Reply:
I've opened https://github.com/buildbot/buildbot/issues/7288 for this comment. This PR does not need to be held up by that.